Dockerfile

```diff
@@ -12,19 +12,93 @@ RUN --mount=type=cache,target=/go/pkg/mod \
     --mount=type=cache,target=/root/.cache/go-build \
     CGO_ENABLED=0 go build -trimpath -ldflags "-s -w" -v -o atlantis .
 
-# Stage 2
-# The runatlantis/atlantis-base is created by docker-base/Dockerfile
-FROM ${ATLANTIS_BASE}:${ATLANTIS_BASE_TAG_DATE}-${ATLANTIS_BASE_TAG_TYPE} AS base
+FROM debian:${DEBIAN_TAG} as deps
 
 # Get the architecture the image is being built for
 ARG TARGETPLATFORM
 
+# Install packages needed for building/verifying dependencies
+# hadolint ignore=DL3008,SC2261
+WORKDIR /tmp/build
+RUN apt-get update \
```

Comment:
Why are these lines repeated on 180? Couldn't we install these packages just once in the debian release?

Reply:
Good point, I can improve this.

Reply:
But now we have 2 places where we need to make sure these dependencies are up to date... Seems like extra maintenance and an easy thing to bump in one section and forget to bump in another